tests/app/submitter/test_convert_payload_0_0_3.py

```diff
@@ -492,7 +492,7 @@ def test_default_answers_not_present_when_not_answered(fake_questionnaire_store)
     ]
 
     output = convert_answers(schema, fake_questionnaire_store, routing_path)
-    data = json.loads(json.dumps(output['data'], for_json=True))
+    data = json.loads(json.dumps(output['data']['answers'], for_json=True))
```

Comment:
As far as I can see, the unit tests still never assert anything about the lists key?

Reply:
We've added two tests to check lists creation: test_list_structure_in_payload_is_as_expected & test_primary_person_not_in_payload_when_not_answered